com.unity.render-pipelines.universal/Shaders/Lit.shader

```diff
@@ -35,7 +35,7 @@ Shader "Universal Render Pipeline/Lit"
         _ClearCoat("Clear Coat", Float) = 0.0
         _ClearCoatMap("Clear Coat Map", 2D) = "white" {}
         _ClearCoatMask("Clear Coat Mask", Range(0.0, 1.0)) = 0.0
-        //_ClearCoatSmoothness("Clear Coat Smoothness", Range(0.0, 1.0)) = 1.0 // TODO: enable
+        _ClearCoatSmoothness("Clear Coat Smoothness", Range(0.0, 1.0)) = 1.0
```

Comment:
This PR re-enables the smoothness parameter for forward too. We should decide how we handle it in all cases.